### PR TITLE
refactor: use execa for install tests, run eslint on samples

### PR DESCRIPTION
**Pull request by JustinBeckwith**
And other misc cleanup.

### CODE REVIEW

File: samples/refreshAccessToken.js [hunk 1]
```diff
@@ -1,112 +0,0 @@
-// Copyright 2017, Google, Inc.
```
Comment (kinwa91) [inline samples/refreshAccessToken.js]: Why was this removed?

Comment (JustinBeckwith) [reply samples/refreshAccessToken.js]: The `refreshAccessToken` method was deprecated in 2.0 :) Users literally never need to call it.
